fidesctl/src/fideslang/models.py

```diff
@@ -353,8 +440,12 @@ class ResourceFilter(BaseModel):
     The ResourceFilter resource model.
     """
 
-    type: str
-    value: str
+    type: str = Field(
+        description="The type of filter to be used (i.e. ignore_resource_arn)",
+    )
+    value: str = Field(
+        description="A string representation of resources to be filtered. Can include wildcards.",
+    )
```

Comment:
@earmenda I took a stab at adding these in, but I think you might be able to improve the verbiage here for me around these (I think similarly with `OrganizationMetadata` and `SystemMetadata`)

Reply:
Can revisit later as needed 👍🏽